baybe/recommenders/meta/base.py

```diff
@@ -30,29 +36,68 @@ def select_recommender(
     ) -> PureRecommender:
         """Select a pure recommender for the given experimentation context.
 
-        Args:
-            batch_size:
-                See :func:`baybe.recommenders.meta.base.MetaRecommender.recommend`.
-            searchspace:
-                See :func:`baybe.recommenders.meta.base.MetaRecommender.recommend`.
-            objective:
-                See :func:`baybe.recommenders.meta.base.MetaRecommender.recommend`.
-            measurements:
-                See :func:`baybe.recommenders.meta.base.MetaRecommender.recommend`.
-
-        Returns:
-            The selected recommender.
+        See :meth:`baybe.recommenders.base.RecommenderProtocol.recommend` for details
```

Comment:
Isn't there a `meta` missing here (compared to the old docstrings)?

Reply:
No, note that I'm pointing to the `Protocol` now, which ultimately is the part that defines the interface. Resolving, but reopen in case of questions